diff --git a/node_modules/send-intent/android/build.gradle b/node_modules/send-intent/android/build.gradle
index 78bb52f..91a73c9 100644
--- a/node_modules/send-intent/android/build.gradle
+++ b/node_modules/send-intent/android/build.gradle
@@ -21,7 +21,7 @@ apply plugin: 'com.android.library'
 
 android {
     namespace "de.mindlib.sendIntent.sendintent"
-    compileSdkVersion 35
+    compileSdkVersion project.hasProperty('compileSdkVersion') ? rootProject.ext.compileSdkVersion : 35
 
     defaultConfig {
         minSdkVersion 23
